### PR TITLE
Add May Meetup

### PR DESCRIPTION
**Pull request by dijital20**
Added Moshe's meetup to the Past Meetups, and added Zvi's to the upcoming meetup.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -12,11 +12,11 @@ of Texas and beyond.
...
+![Zvi Boshernitzan Avatar](./assets/images/ZviBoshernitzan.jpg){: style="height:200px;width:175px" align=left}
 
-#### _Talk_:  Python :heart: Rust - Moshe Zadka
+#### _Talk_:  Chuck Norris vs AI - Zvi Boshernitzan
 
-Introduction to Rust and how to use PyO3 to integrate Python with Rust.
+I'll be sharing my journey through a first project using using text embeddings, which can often be a great way to enhance an unconventional data set with dated references.
 
-_Moshe has been involved in the Linux community since 1998, helping in Linux "installation parties". They have been programming Python since 1999, and has contributed to the core Python interpreter. Moshe has been a DevOps/SRE since before those terms existed, caring deeply about software reliability, build reproducibility and other such things. They have worked in companies as small as three people and as big as tens of thousands -- usually some place around where software meets infrastructure._
+_Zvi Boshernitzan is an software engineer / technologist in austin focused on emerging startups._
```
Comment (matthewcriswell) [inline docs/index.md]: I suggest:
"Zvi Boshernitzan is a software engineer and technologist based in Austin and focuses on emerging startups."

**Review by matthewcriswell — COMMENTED** [review]
Looks great except for some grammar nits I've picked.

File: docs/index.md (hunk 1 above)
Comment (MasonEgger) [inline docs/index.md]: Always recommend using this feature. If y'all like it, one clicky commits it :)
```suggestion
Zvi Boshernitzan is a software engineer and technologist based in Austin and focuses on emerging startups.
```

**Review by MasonEgger — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 1 above)
Comment (toonarmycaptain) [reply docs/index.md]: This is my first time learning of this feature; I love it.

Comment (dijital20) [reply docs/index.md]: I think this was fixed in Mason's comment.

Comment (dijital20) [reply docs/index.md]: Any reason we're not italicizing the text this time? We did previously with Moshe's.

File: docs/index.md [hunk 2]
```diff
@@ -12,11 +12,11 @@ of Texas and beyond.
...
+![Zvi Boshernitzan Avatar](./assets/images/ZviBoshernitzan.jpg){: style="height:200px;width:175px" align=left}
 
-#### _Talk_:  Python :heart: Rust - Moshe Zadka
+#### _Talk_:  Chuck Norris vs AI - Zvi Boshernitzan
 
-Introduction to Rust and how to use PyO3 to integrate Python with Rust.
+I'll be sharing my journey through a first project using using text embeddings, which can often be a great way to enhance an unconventional data set with dated references.
 
-_Moshe has been involved in the Linux community since 1998, helping in Linux "installation parties". They have been programming Python since 1999, and has contributed to the core Python interpreter. Moshe has been a DevOps/SRE since before those terms existed, caring deeply about software reliability, build reproducibility and other such things. They have worked in companies as small as three people and as big as tens of thousands -- usually some place around where software meets infrastructure._
+Zvi Boshernitzan is a software engineer and technologist based in Austin and focuses on emerging startups.
```
Comment (MasonEgger) [inline docs/index.md]: @dijital20 is right. Make italic
```suggestion
_Zvi Boshernitzan is a software engineer and technologist based in Austin and focuses on emerging startups._
```

**Review by MasonEgger — CHANGES_REQUESTED** [review]

**Review by MasonEgger — APPROVED** [review]
LGTM

**Pull request merged**